timesketch/api/v1/resources/sigma.py

```diff
@@ -89,4 +95,72 @@ def get(self, rule_uuid):
             abort(
                 HTTP_STATUS_CODE_NOT_FOUND, 'No sigma rule found with this ID.')
 
-        return return_rule
+        meta = {'current_user': current_user.username,
+                'rules_count': len(sigma_rules)}
+        return jsonify({'objects': return_rule, 'meta': meta})
```

Comment:
so objects here should be `'objects': [return_rule]` as in that should be a list.

Reply:
ack